guides/source/command_line.md

```diff
@@ -376,37 +376,37 @@ $ bin/rails destroy model Oops
       remove      test/fixtures/oops.yml
 ```
 
-Rake
-----
+bin/rails
+---------
 
-Rake is Ruby Make, a standalone Ruby utility that replaces the Unix utility 'make', and uses a 'Rakefile' and `.rake` files to build up a list of tasks. In Rails, Rake is used for common administration tasks, especially sophisticated ones that build off of each other.
+Rake is Ruby Make, a standalone Ruby utility that replaces the Unix utility 'make', and uses a 'Rakefile' and `.rake` files to build up a list of tasks. In Rails, Rake is used for common administration tasks, especially sophisticated ones that build off of each other. Since Rails 5.0+ rake commands implemented in Rails Framework itself. We prefer to use bin/rails instead of rake.
```

Comment:
I feel like it's worth it to keep the mention, but to say that's it's not the recommended way to interact with Rails anymore. I know a lot of people (including myself) who are used to using the Rake commands, and I feel like dumping the whole paragraph might be a bit confusing to end users.